project/src/com/company/transaction/Transaction.java

```diff
@@ -0,0 +1,110 @@
+package com.company.transaction;
+
+import java.util.*;
+
+public class Transaction implements Comparable{
```

Comment:
[Fix] nu folosi Comparable raw, foloseste-l cu generice:  Comparable<Transaction>, vezi ca si Intellij se plange
![image](https://user-images.githubusercontent.com/67580205/161415802-ec6e4bba-391f-4ba3-92b7-18f44b4a2991.png)
Oricand vezi in intelli un cuvant highlightuit asa, sa faci hover findca sigur iti ofera o sugestie de imbunatatire

Reply:
Am rezolvat acest lucru prin a schimba Comparable in Comparable<Transaction>.

Commit: 
https://github.com/AmarKayed/Advanced-OOP-Project/commit/6cf64731dc73781b1a5dbc6d79d3ed097bf12b09